scripts/components/BucketCollections.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import BucketTabs from "./BucketTabs";
+
+
+class DataList extends Component {
+  render() {
+    const {
+      bid,
+      data,
+      listLoaded,
+    } = this.props;
```

Comment:
This can probably fit on a single line. We usually spread on multiple lines when it exceeds 80c.